da/da_test.go

```diff
@@ -309,3 +310,13 @@ func doTestSubmitLargeBlocksOverflow(t *testing.T, dalc *DAClient) {
 	assert.Equal(StatusSuccess, resp.Code, "remaining blocks should submit")
 	assert.EqualValues(resp.SubmittedCount, 1, "submitted count should match")
 }
+
+func doTestRetrieveNoBlocksFound(t *testing.T, dalc *DAClient) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	result := dalc.RetrieveBlocks(ctx, 123)
+	assert.Equal(StatusNotFound, result.Code, "should return not found")
+	assert.Contains(result.Message, "blob: not found")
```

Comment:
nit: can we compare "result.Message" with `ErrBlobNotFound.Error()` here to remove redundancy?

Reply:
Agreed.